docs/current/162770-faq.md

```diff
@@ -2,12 +2,119 @@
 slug: /162770/faq
 ---
 
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
 # FAQ
 
 ### What language SDKs are available for Dagger?
 
 We currently offer technical previews of a [Go SDK](/sdk/go), a [Node.js SDK](/sdk/nodejs) and a [Python SDK](/sdk/python). Waiting for your favorite language to be supported? [Let us know which one](https://airtable.com/shrzABOn1wCk5yBF4), and we'll notify you when it's ready.
 
+### How do I install Dagger?
+
+Refer to the documentation for information on how to install the [Dagger CLI](./cli/465058-install.md) and the Dagger [Go](./sdk/go/371491-install.md), [Node.js](./sdk/nodejs/835948-install.md) and [Python](./sdk/python/866944-install.md) SDKs.
+
+### How do I update Dagger?
+
+:::tip
+[Learn more about compatibility between the Dagger Engine, the Dagger CLI and Dagger SDKs](#what-compatibility-is-there-between-the-dagger-engine-dagger-sdks-and-dagger-cli-versions).
+:::
+
+#### CLI
+
+To update the Dagger CLI, download and uncompress the [latest release of the Dagger CLI](https://github.com/dagger/dagger/releases) for your platform (replace the placeholders with the correct values) and replace your existing Dagger CLI binary with the newer version:
+
+```shell
+wget https://github.com/dagger/dagger/releases/download/vMAJOR.MINOR.PATCH/dagger_vMAJOR.MINOR.PATCH_PLATFORM_ARCHITECTURE.tar.gz
+tar -xzvf dagger_vMAJOR.MINOR.PATCH_PLATFORM_ARCHITECTURE.tar.gz
+sudo mv dagger /usr/local/bin/dagger
+```
```

Comment:
This is the preferred way of installing the Dagger CLI:

```console
curl -L https://dl.dagger.io/dagger/install.sh | sh
```

The script will figure out:
- the latest version
- the appropriate platform
- the appropriate architecture

The outcome is the latest CLI installed at `./bin/dagger`

Reply:
Modified